cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -508,6 +508,60 @@ spec:
   subresources:
     status: {}
 # OLM: END CEPH OBJECT STORE USERS CRD
+# OLM: BEGIN CEPH OBJECT STORE REALMS CRD
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cephobjectrealms.ceph.rook.io
+spec:
+  group: ceph.rook.io
+  names:
+    kind: CephObjectRealm
+    listKind: CephObjectRealmList
+    plural: cephobjectrealms
+    singular: cephobjectrealm
+  scope: Namespaced
+  version: v1
+  subresources:
+    status: {}
+# OLM: END CEPH OBJECT STORE REALMS CRD
+# OLM: BEGIN CEPH OBJECT STORE ZONEGROUPS CRD
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cephobjectzonegroups.ceph.rook.io
+spec:
```

Comment:
Per a discussion with @travisn, he mentioned that since the CR is so simple we don't need a validation section. The ObjectUser doesn't have a validation section either, which is what I based the other CRs off of. Let me know if you still want me to add it